scripts/spack/packages/umpire/package.py

```diff
@@ -21,12 +21,9 @@ class Umpire(CachedCMakePackage, CudaPackage, ROCmPackage):
 
     maintainers = ['davidbeckingsale']
 
-    # TRIBOL EDIT START
-    version('6.0.0tribol', commit='5f474c7501daf365d6015a2141821d83bd799ffa', submodules="True")
```

Comment:
I'm assuming there is no real reason to do it this way, where we have our own tribol version pointing to some commit sha? And per below we can just point to a version of umpire?